src/server/biosecurity/disinfection/index.js

```diff
@@ -1,6 +1,8 @@
-import { QuestionPage } from '~/src/server/common/model/page/question-page-model.js'
```

Comment:
`biosecurity` -> `biosecurity-details`? We have another section called `biosecurity-map` incoming, so we'll end up with a weird ambiguity if we don't call it by its full name

Reply:
I've followed the same convention as in the URLs in the prototype, biosecurity for this one and biosecurity-map for the other. I can rename the stuff but URLs will still be kept as biosecurity. Would you want the section key under which we store the data in this section to also be called biosecurityDetails?

Reply:
ah - that's fine. :) so long as we have a principled way to separate the sections.